resources_pages/install_ds_stack_windows.md

```diff
@@ -3,290 +3,540 @@ layout: page
+
+After the installation is complete, we will add the R executables to the PATH variable in terminal so that your can use it without typing the full path to R each time. Open a terminal and type:
 
 ```
-code --version
+code ~/.bash_profile
 ```
 
-you should see something like this if you were successful:
+Append the following line to the file
+
 ```
-1.36.1
-2213894ea0415ee8c85c5eea0d0ff81ecc191529
-x64
+# Add R and Rscript to PATH
+export PATH="/c/Program Files/R/R-4.0.2/bin/x64":$PATH
 ```
 
-### Customizing VS Code with Extensions
+> Note: Although it is possible to install R through Anaconda, we highly recommend not doing so. In case you have already installed R using Anaconda you can remove it by executing `conda uninstall r-base`.
 
-The real magic of VS Code is in the extensions that let you add languages, debuggers, and tools to your installation to support your specific workflow. From within VS Code you can open up the [Extension Marketplace](https://code.visualstudio.com/docs/editor/extension-gallery) to browse and install extensions by clicking on the Extensions icon in the Activity Bar indicated in the figure below.
+### RStudio
 
-![](/resources_pages/imgs/vscode.png)
+Download the Windows version of RStudio from <https://www.rstudio.com/products/rstudio/download/preview>. Open the file and follow the installer instructions.
 
-To install an extension, you simply search for it in the search bar, click the extension you want, and then click "Install". There are extensions available to make almost any workflow or task you are interested in more efficient! Here we are interested in setting up VS Code as a Python IDE. To do this, search for and install the following extensions:
+To see if you were successful, try opening RStudio by clicking on its icon. It should open and looks something like this picture below:
 
-- Python (everything Python: notebooks, debugging, linting, formatting, etc.)
-- markdownlint (markdown linting and style checking extension)
-- GitLens (powerful extension that extends VS Code's native git capabilities)
-- Docker (easily use Docker from VS Code)
-- (Optional) Material Theme and/or Predawn Theme Kit (additional colour themes to choose from)
-- (Optional) Material Icon Theme (great-looking custom file icons!)
-- (Optional) Bracket Pair Colorizer 2 (add colour to help distinguish your brackets: (), [], {})
+![](/resources_pages/imgs/RStudio.png)
 
-This [video tutorial](https://www.youtube.com/watch?v=-nh9rCzPJ20) is an excellent introduction to using VS Code in Python.
+By default, RStudio installs packages to `C:/Users/your_username/Documents/R/win-library/4.0`, but when it is run directly from the terminal it instead uses  `C:/Users/your_username/R/win-library/4.0`. To make sure that R from terminal uses the same package directory as R studio, we will edit `~/.Rprofile` by typing:
+
+```
+code ~/.Rprofile
```

Comment:
The amount of hours it took me to figure out where to put what here shall remain unnamed. Do you think this is the best solution? I found so many different solutions here, including editing `R_LIBS_PATH`, .Rprofile at two places, .Renviron at two places. The one I suggested worked the most robustly of the ones I tried and is also the most alike how they are editing e.g. `.bash_profile` so I thought what they would learn there would transfer to here (and this also serve as a way of solidifying this concept of editing profile files.

Reply:
We should be setting `R_LIBS` in `.Renviron` not `.Rprofile` as suggested here: https://github.ubc.ca/MDS-2019-20/DSCI_522_dsci-workflows_students/issues/4

In general, these two main files serve these two functions: 
- `.Renviron` - which contains environment variables to be set in R sessions.
- `.Rprofile` - which contains R code to be run in each session.

(from @jennybc's [What They Forgot to Teach You About R](https://rstats.wtf/index.html) book). More can be found on these two files in this chapter: https://rstats.wtf/r-startup.html#renviron

Reply:
Great, thank you! I will update this.

Reply:
One more thing I realized as I was going to make this change on my machine. The [Rtools website recommend doing the PATH manipulations in `.Rprofile`](https://cran.r-project.org/bin/windows/Rtools/), do you think it will be confusing for students that we are modifying env paths in two different files? And is PATH a special case that cannot be set in `.Renviron`, or are they technically doing modifying the wrong file?